com.unity.multiplayer.mlapi/Runtime/Messaging/InternalMessageHandler.cs

```diff
@@ -380,7 +380,9 @@ public void HandleNetworkLog(ulong clientId, Stream stream)
         {
             using (var reader = PooledNetworkReader.Get(stream))
             {
+                var length = stream.Length;
                 var logType = (NetworkLog.LogType)reader.ReadByte();
+                NetworkManager.Singleton.NetworkMetrics.TrackServerLogReceived(clientId, (uint)logType, (ulong)length);
```

Comment:
I believe you're not supposed to use the Singleton for this class anymore

Reply:
Oh! I will remove references to it, that was probably out of habit rather than me thinking about it intentionally 😆